lib/starknet_explorer/message.ex

```diff
@@ -0,0 +1,78 @@
+defmodule StarknetExplorer.Message do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias StarknetExplorer.Repo
+
+  @primary_key {:message_hash, :string, autogenerate: false}
+  @fields [
+    :from_address,
+    :to_address,
+    :payload,
+    :type,
+    :transaction_hash,
+    :message_hash,
+    :timestamp,
+    :network
+  ]
+
+  schema "messages" do
+    field :from_address, :string
+    field :transaction_hash, :string
+    field :to_address, :string
+    field :payload, {:array, :string}
+    field :timestamp, :integer
+    field :type, :string
+    field :network, Ecto.Enum, values: [:mainnet, :testnet, :testnet2]
+    # belongs_to :transaction, Transaction, foreign_key: :transaction_hash, references: :hash
```

Comment:
I was not able to get this relation to work correctly but in order to be unblocked I skipped it for now